crap-advisor-frontend/src/core/components/RestroomMarker/index.css

```diff
@@ -0,0 +1,44 @@
+.marker-container {
+    position: relative;
+    top: -15px;
+    left: -15px;
+    width: fit-content;
```

Comment:
Woah, "fit-content"? So ludicrous choise

Reply:
Man i was bruteforcing lexems and this is first that worked :|
what should i use?

Reply:
First, try 100%. If it's doenst work well, you can write correct pixels number.

Reply:
100% doesn't work, changed to 40px